iterative/utils/helpers.go

```diff
@@ -7,6 +7,17 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+var (
+	AWSSynthRegions = map[string]map[string]string{
```

Comment:
Given that it's only for AWS, wouldn't it be enough to use a `map[string]string`? Else, should it be named `SynthRegions` instead?

Reply:
Indeed probably I would keep it in the AWS provider instead of utils

Reply:
https://github.com/iterative/terraform-provider-iterative/pull/344#issuecomment-1005009172

Reply:
I can roll it back more, I left it in to inflict my feelings that these region mappings should all be defined in one place.

Reply:
Probably makes sense to have it in the provider following the same pattern as the others. Also as @0x2b3bfa0 pointed out there is not need to use a matrix